### ISSUE TITLE
Questionnaire markdown file is not actually convenient

### ISSUE THREAD

**Issue by johannhof** [w3ctag/security-questionnaire#172]
https://www.w3.org/TR/security-privacy-questionnaire/#reviews says

> When requesting a [review](https://github.com/w3ctag/design-reviews) from the [Technical Architecture Group (TAG)](https://www.w3.org/2001/tag/), please provide the TAG with answers to the questions in this document. [This Markdown template](https://raw.githubusercontent.com/w3ctag/security-questionnaire/main/questionnaire.markdown) may be useful when doing so.

The page that is links to is https://raw.githubusercontent.com/w3ctag/security-questionnaire/main/questionnaire.markdown, which is the raw version of https://github.com/w3ctag/security-questionnaire/blob/main/questionnaire.markdown

It says that the markdown version of the questions are added "for your convenience", but neither of these files is particularly convenient and forces me to do a bunch of manual work anyway. With the raw file, I have to manually remove all the markdown quotes. With the rendered file, GitHub doesn't allow me to copy the numbers of the list items so that I would have to add them.

My suggestion would be to remove the quotation `>`s so that the list can be copied from the raw file.

**Comment by domenic** [comment]
I'm confused by this request. When quoting another document (namely the questionnaire), why wouldn't the `>` prefix be appropriate?

You can see an example that keeps the quotations at https://github.com/WICG/translation-api/blob/main/security-privacy-questionnaire.md . I think it looks fine.

**Comment by johannhof** [comment]
Yeah, I think I misunderstood how it was supposed to be used. I thought just the text inside of the quotes was the template, and didn't consider using it *with* the quotes so that it looks like the file you linked.

That was probably just me being dumb, so with that misunderstanding clarified I don't have strong feelings on merging #177 vs. closing this issue.
